src/current/_includes/v24.2/install-docker-steps.md

```diff
@@ -0,0 +1,57 @@
+{% comment %}This include is used in install-cockroachdb-*.md{% endcomment %}
+{% capture deployment_link %}
+{% if page.name contains "mac" %}[Deploy a local container in Docker]({% link {{ page.version.version }}/start-a-local-cluster-in-docker-mac.md %})
+{% elsif page.name contains "windows" %}[Deploy a local container in Docker]({% link {{ page.version.version }}/start-a-local-cluster-in-docker-windows.md %})
+{% else %}[Deploy a local container in Docker]({% link {{ page.version.version }}/start-a-local-cluster-in-docker-linux.md %})
+{% endif %}
+{% endcapture %}
+
+{{site.data.alerts.callout_danger}}
+Running a stateful application like CockroachDB in Docker is more complex and error-prone than most uses of Docker. Unless you are very experienced with Docker, we recommend starting with a different installation and deployment method.
+{{site.data.alerts.end}}
+
+CockroachDB's Docker images are [multi-platform images](https://docs.docker.com/build/building/multi-platform/) that contain binaries for both Intel and ARM. Multi-platform images do not take up additional space on your Docker host.
+
+Experimental images are not qualified for production use and not eligible for support or uptime SLA commitments.
+
+1. Install a container runtime, such as [Docker Desktop](https://docs.docker.com/desktop/install/).
```

Comment:
the link to Docker Desktop is 404ing

<img width="1154" alt="Screenshot 2024-08-20 at 11 15 30 AM" src="https://github.com/user-attachments/assets/5d49b800-8a3f-4b42-97fc-e5cfe28330c9">